src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,175 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
```

Comment:
I have a good impression looking through your code. Like it! Well done!

Reply:
Thank you, constructive comments make my eyes wider every time)